tests/page-objects/EditorPage.ts

```diff
@@ -37,24 +37,43 @@ export class EditorPage {
         await this.addOpposingReviewer();
     }
 
+    private async addEditorsToPeoplePicker(picker: Selector): Promise<void> {
+        await t.expect(picker.find('.selected_people_list__item').count).eql(1);
+        const addButton = picker.find('.pod__button');
+        await t.click(addButton);
+        await this.assertPeoplePickerSearch();
+        await t.expect(Selector('.typography__heading--h2').visible).ok();
+        await t.expect(Selector('#peoplePickerSearch').visible).ok();
+        const firstAddButton = Selector('.people-picker__modal_list--item:nth-child(odd) .pod__button');
+        await t.expect(firstAddButton.visible).ok();
+        await t.click(firstAddButton);
+        const secondAddButton = Selector('.people-picker__modal_list--item:nth-child(even) .pod__button');
+        await t.expect(secondAddButton.visible).ok();
+        await t.click(secondAddButton);
+        await t.expect(Selector('.people-picker__selected-tabs').child('.people-picker__selected-tab').count).eql(2);
+        await t.click(Selector('.modal__buttons_container').find('.button--primary'));
+        await t.expect(Selector('.modal .modal__fullscreen').exists).eql(false);
+        await t.expect(picker.find('.selected_people_list__item').count).eql(3);
+    }
+
```

Comment:
Can this not be abstracted as its a clone of the addPersonToPeoplePicker function.

Reply:
Addressed